test/cloudant.test.js

```diff
@@ -47,9 +47,13 @@ describe('cloudant connector', function() {
         ],
       },
     });
-    Product.destroyAll(function(err) {
-      CustomerSimple.destroyAll(function(err) {
-        done();
+
+    db.automigrate(function(err) {
```

Comment:
Regarding my last comment, we should add a function name here to make it consistent.